packages/api/src/routes/medical/internal-mpi.ts

```diff
@@ -42,7 +44,17 @@ router.get(
         lastNameInitial,
       },
     });
-    return res.status(status.OK).json(foundPatients.map(dtoFromModel));
+    const patientsWithUniqueId = foundPatients.map(patient => {
+      const uniqueId = createPatientUniqueId(patient.cxId, patient.id);
+      const updatedPatient: Patient = {
+        ...patient,
+        id: uniqueId,
```

Comment:
the issue is we the docIds needs to be made of the cxId and patientId in order to support the flow we have with retrieving metadata files from s3. PatientDTO in api-sdk understandably does not return cxID, so the best place to do this is here. I think this is not worth changing since odds are we will move domain to core before we reuse this endpoint for anything else, and then we wont need the endpoint at all since the lambda can read from db directly and all this code will be refactored.

I can't think of any upcoming situations where we will resuse this endpoint in the near future I think this is the best solution for now